varcode/effect_ordering.py

```diff
@@ -52,10 +52,18 @@ def top_priority_transcript_effect(effects):
         elif priority == best_priority:
             best_effects.append(effect)
 
-    def effect_cds_len(effect):
-        """
-        Returns length of coding sequence of transcript associated with effect
-        """
-        return len(effect.transcript.coding_sequence)
-
-    return max(best_effects, key=effect_cds_len)
\ No newline at end of file
+    if any(effect.transcript.complete for effect in best_effects):
```

Comment:
...but why leave in incomplete transcripts if we know for sure we're not going to use them? We can't compare their CDS length, so the key would have to be something like:
```
def key_fn(effect): 
   c = effect.transcript.complete
   cds_len = len(effect.transcript.coding_sequence) if c else 0
   return c, cds_len
```

I find that less clear than explicitly dropping the incomplete terms. 

If we wanted to unify both branches we'd need something like:

```
def key_fn(effect): 
   c = effect.transcript.complete
   cds_len = len(effect.transcript.coding_sequence) if c else 0
   return c, cds_len, len(transcript)
```

Which is more elegant but I think less clear on what's happening.